community/rfcs/24-05-24-001-OPEA-Code-Structure.md

```diff
@@ -0,0 +1,66 @@
+**Author**
+
+[ftian1](https://github.com/ftian1), [lvliang-intel](https://github.com/lvliang-intel), [hshen14](https://github.com/hshen14)
+
+**Status**
+
+Under Review
+
+**Objective**
+
+Define a clear criteria and rule of adding new codes into OPEA projects.
+
+**Motivation**
+
+OPEA project consists of serveral repos, including GenAIExamples, GenAIInfra, GenAICompos, and so on. We need a clear definition on where the new code for a given feature should be put into for a consistent and well-orgnized code structure.
```

Comment:
organized

Reply:
thanks for the reminder. looks like the preci doesn't include spelling check. I will sync with pre-ci owner as well.